### DISCUSSION TITLE
Any chance the sizes for the icons, banners, etc. can be standardized with steam?

### DISCUSSION THREAD

**Discussion by fxzzi** [lutris/lutris#4893] [Ideas]
This will allow us to use amazing resources like steamgriddb to get our art which will make lutris look much nicer :)

**Comment by strycore** [comment]
The icons and banner originally come straight out of Steam. 
The cover art format we use is from IGDB which uses a format just a bit different from the one from Steam / SteamGridDB.
Since the ratio of the of IGDB covers is not 100% consistent, we may be able to mix both without much of a difference.

On my installation, I make sure to not leave blank covers like this:

![image](https://github.com/lutris/lutris/assets/280336/5b2bbad2-791d-457f-aeb2-f306b9181d1f)

This often requires fiddling a bit on the website end, merging games or creating aliases. I will eventually add something accessible to non moderators to request game merging.
